### PR TITLE
feat: test deployments for `SwapAndBridge`

### PR DESCRIPTION
**Pull request by dohaki**
Add deployment scripts `SwapAndBridge` contracts for 1inch and Uniswap. Also deployment files for QA setup

### CODE REVIEW

File: deploy/consts.ts [hunk 1]
```diff
@@ -141,13 +141,17 @@ export const L2_ADDRESS_MAP: { [key: number]: { [contractName: string]: string }
     l2Usdc: "0xaf88d065e77c8cC2239327C5EDb3A432268e5831",
     cctpTokenMessenger: "0x19330d10D9Cc8751218eaf51E8885D058642E08A",
     cctpMessageTransmitter: "0xC30362313FBBA5cf9163F0bb16a0e01f01A896ca",
+    uniswapV3SwapRouter: "0xE592427A0AEce92De3Edee1F18E0157C05861564",
+    "1inchV6Router": "0x111111125421ca6dc452d289314280a0f8842a65",
```
Comment (james-a-morris) [inline deploy/consts.ts:145]: Is this checksummed?

**Review by james-a-morris — COMMENTED** [review]
One nit

**Review by mrice32 — APPROVED** [review]
LGTM!

File: deploy/consts.ts (hunk 1 above)
Comment (dohaki) [reply deploy/consts.ts]: Checksummed now

**Pull request merged**